eitprocessing/eit_data/sentec.py

```diff
@@ -18,4 +34,144 @@ def _from_path(  # pylint: disable=too-many-arguments
         max_frames: int | None = None,
         return_non_eit_data: bool = False,
     ) -> Self | tuple[Self, ContinuousDataCollection, SparseDataCollection]:
-        ...
+
+        with open(path, "br") as fh:
+            # Find the length of the file
+            fh.seek(0, 2)
+            file_length = fh.tell()
+
+            # go back to the beginning of the file
+            fh.seek(0, 0)
+
+            # instantiate reader
+            reader = Reader(fh)
+
+            # read the version int8
+            version = reader.unsigned_char('little')
+
+            if version < 2:
+                warnings.warn(f'File version {version}. Version 2 or higher expected.')
+
+            time = []
```

Comment:
As far as I can see `time` is a list of integers. I expect the time to be
floating point numbers. Is there a conversion that should be done?

Reply:
The timestamp is a uint64 in Matlab, so it is a list of integers. I think that the value is in milliseconds. I don't know what is considered the 0, though.

Reply:
I think we should convert this to seconds. `time` is not the same as a `timestamp`, and always expected to be in the same units (seconds). We should be able to check the number by comparing the number of frames, the framerate and the timestamps.